tests/cupy_tests/linalg_tests/test_product.py

```diff
@@ -256,14 +258,16 @@ def test_transposed_tensordot(self, xp, dtype):
         b = testing.shaped_arange((4, 3, 2), xp, dtype).transpose(2, 0, 1)
         return xp.tensordot(a, b)
 
-    @testing.for_all_dtypes()
+    @testing.for_all_dtypes(excludes=(numpy.uint8, numpy.int8, numpy.uint16,
```

Comment:
Good point. Those types need to be excluded because the following tensordot result has the maximum value of 358190, but it apparently exceed the limit of both int8 and int16. So if we don't exclude these types, this test case just results in checking that overflowed values are the same between numpy and cupy. This makes no sense.